### ISSUE TITLE
change dynamically `ModelView` properties?

### ISSUE THREAD

**Issue by qbait** [airbnb/epoxy#919]
I have 2 rows that are different only because of `@ModelView` annotation
1) 
```@ModelView(autoLayout = ModelView.Size.MATCH_WIDTH_MATCH_HEIGHT, fullSpan = false)
class SampleRow1 @JvmOverloads constructor(...){...}```

2) 
```@ModelView(autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT)`
class SampleRow2 @JvmOverloads constructor(...){...}```

Is there a possibility to have a single ModelView for that?
I'm using `Paris` to style views inside my ModelViews. Is there a possibility to style also the whole ModelView by it?

**Comment by elihart** [comment]
yes, if you are using Paris you don't need to use `autoLayout`. You can change the top level view heigh and width layout properties with Paris

**Comment by premacck** [comment]
@elihart But what if we're not using Paris?
I have actually encountered a same issue where I'm using the same `@ModelView(autoLayout = ModelView.Size.WRAP_WIDTH_WRAP_HEIGHT)` class in two places, because I have to display same layout in a carousel, as well as a vertical list.
But the items in the vertical list appear to be wrapped in height, which looks ugly.

So to solve that for now, I've made an empty subclass of the above layout class with `@ModelView(autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT)` and used that class for the vertical list, but there is code duplication because of that. Because both of these classes have different generated `ModelBuilder` classes, I couldn't make the `@ModelProp` methods usage common.

Can you please provide any better solution to this issue? Maybe something that doesn't involve making an extra class?
